documentation/src/main/asciidoc/stories/assembly_migrating_cache_stores.adoc

```diff
@@ -1,8 +1,43 @@
 [id='cache_store_migration']
 :context: configure
-= Migrating Cache Stores Between {brandname} Versions
+= Migrating Data Between {brandname} Versions
+Upgrading to the latest {brandname} version typically involves creating new
+clusters with the target version running in parallel to the version from which
+you are migrating. You can then create a shared persistent cache store between
```

Comment:
Should we link to our rolling upgrade docs here?

Reply:
tbh I'd rather reuse the content as part of an upgrading guide. essentially we'd have the same content under Configuration > Persistence and Upgrading. I'll set this up and show you. explaining here might not be clear.